data/World/Fire Temple.json

```diff
@@ -34,6 +36,12 @@
         "locations": {
             "Fire Temple Big Lava Room Lower Open Door Chest": "True",
             "Fire Temple Big Lava Room Blocked Door Chest": "is_adult and has_explosives",
+            "Fire Temple Elevator Room Recovery Heart 1": "(Small_Key_Fire_Temple, 3) and can_use(Goron_Tunic)",
+            "Fire Temple Elevator Room Recovery Heart 2": "(Small_Key_Fire_Temple, 3) and can_use(Goron_Tunic)",
+            "Fire Temple Elevator Room Recovery Heart 3": "(Small_Key_Fire_Temple, 3) and can_use(Goron_Tunic)",
```

Comment:
Adult with the Hookshot might find these recovery hearts safe under FTR. At the least with Longshot is actually seems kind of reasonable. In the past I have resisted including hookshot though just to save a small amount of time on the heat timer. I think I would rather always just require the tunic than bring in the hookshot for that. You might want to mess around with it a bit and see how you feel about how far you can get in 24s. (Generally we don't take advantage of the "it's hot in here" text.)

Reply:
I have been unsuccessful getting these items in 24s without hookshot. I think they should remain as is.

Reply:
With longshot in particular it's actually pretty doable. But again not a big fan of the ____shots  being used just to save time on the heat timer. I've asked for somebody's opinion who's done a lot of work with heat timer logic in the past, and we'll see how they make out.